doc/lua_api.txt

```diff
@@ -2039,9 +2039,19 @@ and `minetest.auth_reload` call the authetification handler.
     * `pos1` and `pos2` are optional and default to mapchunk minp and maxp.
 * `minetest.clear_objects()`
     * clear all objects in the environments
-* `minetest.emerge_area(pos1, pos2)`
-    * queues all mapblocks in the area from pos1 to pos2, inclusive, for emerge
-    * i.e. asynchronously loads blocks from disk, or if inexistent, generates them
+* `minetest.emerge_area(pos1, pos2, callback, param)`
+    * Queue all blocks in the area from `pos1` to `pos2`, inclusive, to be asynchronously
+    * fetched from memory, loaded from disk, or if inexistent, generates them.
+    * If `callback` is a valid Lua function, this will be called for each block emerged.
```

Comment:
Perhaps say "processed" instead of "emerged"? The function gets called too if it didn't emerge.